_posts/2019-05-29-Student-SOCIS-2019.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: SOCIS student
```

Comment:
A more informative title maybe?

Reply:
Some suggestion?

Reply:
Maybe "Here's to Summer of Code with EPY" ?